--- AIS_DataMapOfSelStat.hxx	2018-02-02 19:22:37.000000000 +0200
+++ AIS_DataMapOfSelStat.hxx	2019-10-02 22:12:51.000000000 +0200
@@ -15,15 +15,15 @@
 // commercial license or contractual agreement.
 
 #ifndef AIS_DataMapOfSelStat_HeaderFile
 #define AIS_DataMapOfSelStat_HeaderFile
 
 #include <SelectMgr_SelectableObject.hxx>
-#include <AIS_LocalStatus.hxx>
+//#include <AIS_LocalStatus.hxx>
 #include <TColStd_MapTransientHasher.hxx>
 #include <NCollection_DataMap.hxx>
 
-typedef NCollection_DataMap<Handle(SelectMgr_SelectableObject),Handle(AIS_LocalStatus),TColStd_MapTransientHasher> AIS_DataMapOfSelStat;
-typedef NCollection_DataMap<Handle(SelectMgr_SelectableObject),Handle(AIS_LocalStatus),TColStd_MapTransientHasher>::Iterator AIS_DataMapIteratorOfDataMapOfSelStat;
+//typedef NCollection_DataMap<Handle(SelectMgr_SelectableObject),Handle(AIS_LocalStatus),TColStd_MapTransientHasher> AIS_DataMapOfSelStat;
+//typedef NCollection_DataMap<Handle(SelectMgr_SelectableObject),Handle(AIS_LocalStatus),TColStd_MapTransientHasher>::Iterator AIS_DataMapIteratorOfDataMapOfSelStat;
 
 
 #endif
